tests/homePage/HomePage.test.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from '@playwright/test';
+
+/**
+ * Test that logo shows up
+ */
+
+test('Should display Logo', async ({ page }) => {
+    await page.goto("https://techstartucalgary.com");
```

Comment:
There should be a helper function in the /tests folder to figure out what env we're on and navigate to the correct page based on dev or prd environment.

Consequently that raises the question on how we should integrate these tests within our CI.

Reply:
I created an env variable called `STAGING`. It's a toggle between 0 and 1. 1 for local env and 0 for prod.
Im running the test on `STAGING =1` And all the tests passed for Chrome, but on Firefox the same join button on the apply page is having an issue. My assumption is that it has to do with the animation on the button.

<img width="959" alt="image" src="https://github.com/user-attachments/assets/cccb2a64-575a-497c-b5a6-1541c3cc6d8c">
<img width="959" alt="image" src="https://github.com/user-attachments/assets/2a53d9f4-2f56-41e6-954f-6aad3c554499">